snapshot-labs/sx-monorepo
# feat: show proposal start and end date in proposal editor

### Summary

Show the expected proposal start and end date in the proposal creation page

Closes: #71 

### How to test

1. Go to the proposal creation page
2. It should display the expected proposal start and end date like the screenshot 

![Screenshot 2024-03-19 at 10 47 02](https://github.com/snapshot-labs/sx-monorepo/assets/495709/461d0823-d477-43c7-9ad9-eff5856dfb5c)

On small screen

![Screenshot 2024-03-19 at 11 29 55](https://github.com/snapshot-labs/sx-monorepo/assets/495709/d10a45a4-3424-485a-9a17-04a6ef5a1f8a)


These date should depends on your space settings. One thing to note is that final time will not be exactly the same, as the start and end date are set only on form submission.

UI from figma
